frog/imports/ui/GraphEditor/SidePanel/ActivityPanel.js

```diff
@@ -83,6 +83,7 @@ class ChooseActivityTypeComp extends Component {
               </div>
             : filteredList.map(x =>
                 <ListComponent
+                  hasPreview={x.meta.exampleData !== undefined}
```

Comment:
In the issue you also mentioned newly created activity, since you changed the template in #355 , it won't be undefined no ?